lib/internal/worker.js

```diff
@@ -459,6 +459,17 @@ class Worker extends EventEmitter {
       };
     });
   }
+
+  getHeapStatistics() {
+    const taker = this[kHandle]?.getHeapStatistics();
```

Comment:
Can you add a typings folder change for this method?

Reply:
Done